docs/samples/storage/storageSpec/README.md

```diff
@@ -0,0 +1,103 @@
+# Predict on a InferenceService with StorageSpec
+## Setup
+1. Your ~/.kube/config should point to a cluster with [KServe installed](https://github.com/kserve/kserve#installation).
+2. Your cluster's Istio Ingress gateway must be [network accessible](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/).
+3. Your cluster's Istio Egresss gateway must [allow accessing S3 Storage](https://knative.dev/docs/serving/outbound-network-access/)
+
+## Create Common Storage Secret
+Create a common secret with your [storage credential](https://console.aws.amazon.com/iam/home#/users), `KServe` reads the secret key to inject
```

Comment:
User may still have multiple secrets, how do we specify that on the spec?

Reply:
In the next phase we can have let kserve controller or create a new controller to sync multiple secrets into one common secret in each namespace. The reason we want to have a common secret so that when using modelmesh or multi-model serving, the model serving pod doesn't have to roll out again to mount on new secret volume.